sdk/schemaregistry/azure-schemaregistry/README.md

```diff
@@ -345,7 +355,7 @@ Integration with [Event Hubs][eventhubs_repo] to send an `EventData` object with
 import os
 from azure.eventhub import EventHubProducerClient, EventData
 from azure.schemaregistry import SchemaRegistryClient
-from azure.schemaregistry.encoder.jsonencoder import JsonSchemaEncoder, JsonSchemaDraftIdentifier
```

Comment:
just checking, `JsonSchemaDraftIdentifier` this was only for the beta and not previously GA?